client/signup/config/flows-pure.js

```diff
@@ -452,10 +452,11 @@ export function generateFlows( {
 		{
 			name: 'woocommerce-install',
 			pageTitle: translate( 'Add WooCommerce to your site' ),
-			steps: [ 'confirm', 'transfer', 'install', 'complete' ],
+			steps: [ 'select-site', 'confirm', 'transfer' ],
 			destination: '/',
 			description: 'Onboarding and installation flow for woocommerce on all plans.',
 			providesDependenciesInQuery: [ 'siteSlug' ],
+			optionalDependenciesInQuery: [ 'siteSlug' ],
 			disallowResume: true,
 			lastModified: '2021-11-11',
```

Comment:
should we update this date?

Reply:
I'm not really sure what this is used for, I think its just to be able to compare signup metric changes against flow changes (e.g. audit when things go wrong), it may also be sent with recordEvent? We can probably ignore this for now and update it before launch. (Added a ticket to the board)